matter/wifi/rs911x/hal/rsi_hal_mcu_timer.c

```diff
@@ -119,8 +119,11 @@ int32_t rsi_timer_start(uint8_t timer_node, uint8_t mode, uint8_t type, uint32_t
                             ((mode == RSI_HAL_TIMER_TYPE_SINGLE_SHOT) ? pdFALSE : pdTRUE),
                             NULL,
                             timer_cb);
-  if (tp->handle == HANDLE0)
+
+  if (tp->handle == HANDLE0) {
```

Comment:
If handle is a pointer, compare with `NULL`

Reply:
done